torch/onnx/symbolic_opset9.py

```diff
@@ -3483,19 +3461,21 @@ def tensor(
     dtype = symbolic_helper._get_const(dtype, "i", "dtype")
     if symbolic_helper._is_packed_list(data):
         if dtype is None:
-            scalar_name = symbolic_helper._unpack_list(data)[0].type().scalarType()
-            dtype = _type_utils.JitScalarType.from_name(scalar_name)
+            dtype = _type_utils.JitScalarType.from_value(
```

Comment:
Should dtype be a JitScalarType object?

Reply:
I didn't understand the question. 

The new change `dtype = JitScalarType.from_value()` creates the same type as the existing `dtype = JitScalarType.from_name()`

`dtype` is the same object as before.

Reply:
I think I saw a call to `.scalar_name()`?

Reply:
No more coffee for today :)